cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,139 @@
+  - [The Power of a Great Ticket](#the-power-of-a-great-ticket)
+- [How We Prioritize Issues](#how-we-prioritize-issues)
+  - [Cypress Relative Priority Score](#cypress-relative-priority-score)
+  - [Cypress Relative Priority to Effort Score](#cypress-relative-priority-to-effort-score)
+- [FAQs](#faqs)
+  - [Why Isn't Anyone Working on My Issue?](#why-isnt-anyone-working-on-my-issue)
+  - [Why Did You Close My Ticket?](#why-did-you-close-my-ticket)
+  - [Backpatching Strategy/Limitations](#backpatching-strategylimitations)
+- [How You Can Help](#how-you-can-help)
+
+
+## The Cypress App Priorities
+
+The Cypress app is constantly looking to improve and grow our testing platform to meet the ever-growing needs of the testing world. Like any software platform, we must balance research, development, and maintenance against real-world resource constraints. We feel that transparency about our future plans will help us grow a stronger relationship with end users who understand that all choices involve trade-offs.
+
+## The Cypress App Roadmap
+
+[The Cypress App Priorities](https://github.com/orgs/cypress-io/projects/13/views/1) board shows the high-level roadmap at Cypress. 
+
+The board flows from left to right as a project moves from the ideation and feasibility phase all the way to General Availability (GA) release. 
+
+Here is a guide to what each column on the board represents:
+
+| Column Name                           | Description            | 
+ | :------------------------------------ | :---------------------- | 
+ | **Under Consideration**               | These tickets are currently being discussed for future work. We are investigating the complexity of the change, the amount of resources necessary to complete it, and how it aligns with other goals for the platform. It is important to note that **not all** items in this column make their way into our scheduled work. It is possible that, upon investigation, a ticket falls in priority due to other goals or constraints.               | 
+| **Planned** |  These are the tickets we are planning to work on, but the work has not yet started. |
+| **In Progress** | These tickets are actively being worked on. |
+| **Experimental** | If a ticket involves a substantial refactoring of the codebase, is large in scope or is a complex feature, we often release them in a turned-off state. End users can opt in to experiment with these changes via feature flags. Work is still ongoing, but we feel it is important to get this feature in users' hands. Feedback is always welcome, but it is especially helpful during this phase of the development cycle. Not all issues are released under the experimental flag before GA. |
+| **Released** | Once an issue has been fully implemented, tested, and bundled into an official release it moves to this column. Features in this column are no longer experimental, feature flags are removed and default behaviors are implemented. |
```

Comment:
Can we also define "Beta" and "Alpha" here? Component testing has frameworks in Alpha, what does that mean relative to these categorizations? @baus

Reply:
Is that information currently captured on the priority board?  I don't see it there but maybe I am just not seeing it.  If it isn't being shown in v1 I would suggest we save that for a later revision.  Scope creep is starting to add up and delay us getting this rolled out.